onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -665,6 +665,7 @@ def testInvalidSessionOptionsConfigEntry(self):
             'SessionOptions does not have configuration with key: ' + invalide_key in str(context.exception))
 
     def testRegisterCustomOpsLibrary(self):
+        return
```

Comment:
Please don't forget to remove this line (This basically doesn't run the test). Please ignore the test failure on your machine - I think I know why it is failing - the script needs to be run from the binaries folder (i.e.) where the custom_op_library.dll exists and when run from some other path, it fails to find it - the logistics of looking for the custom_op_library.dll can be improved in the test so that it is more robust. When this script is run as part of build.py, on my machine and on CI machines - it passes.

Reply:
Thank you @hariharans29. Forget to ignore this file for uploading...